File: src/main/java/com/teamiurg/techexpansion/item/TEItems.java
package com.teamiurg.techexpansion.item;

import com.teamiurg.techexpansion.TechExpansion;
import com.teamiurg.techexpansion.engine.render.RenderHandler;
import com.teamiurg.techexpansion.item.tools.TETools;

import net.minecraft.item.Item;
import net.minecraftforge.fml.common.registry.GameRegistry;

public class TEItems {
	
	/* ingots */
	public static Item copper_ingot = new TEItem("copper_ingot", TechExpansion.tab_techExpansion, 64);
	public static Item aluminum_ingot = new TEItem("aluminum_ingot", TechExpansion.tab_techExpansion, 64);
	public static Item tin_ingot = new TEItem("tin_ingot", TechExpansion.tab_techExpansion, 64);
	
	public static void registerItems() {
		/* ingots */
		GameRegistry.registerItem(copper_ingot, "copper_ingot");
		GameRegistry.registerItem(aluminum_ingot, "aluminum_ingot");
		GameRegistry.registerItem(tin_ingot, "tin_ingot");
		TETools.registerItems();
	}

	public static void registerRenders() {
		/* ingots */
		RenderHandler.registerRender(copper_ingot);
		RenderHandler.registerRender(aluminum_ingot);
		RenderHandler.registerRender(tin_ingot);
		TETools.registerRenders();
	}
}

File: src/main/scala/com/teamisotope/techexpansion/item/TEItems.scala
package com.teamisotope.techexpansion.item

import com.teamisotope.techexpansion.TechExpansion
import com.teamisotope.techexpansion.item.special.CoFHTest
import net.minecraftforge.fml.common.registry.GameRegistry
import com.teamisotope.techexpansion.util._
import net.minecraftforge.oredict.OreDictionary

object TEItems {

  val copper_ingot: TEItem = new TEItem("copper_ingot", TechExpansion.tab_resources, 64)
  val aluminum_ingot: TEItem = new TEItem("aluminum_ingot", TechExpansion.tab_resources, 64)
  val tin_ingot: TEItem = new TEItem("tin_ingot", TechExpansion.tab_resources, 64)
  val nickel_ingot: TEItem = new TEItem("nickel_ingot", TechExpansion.tab_resources, 64)
  val lead_ingot: TEItem = new TEItem("lead_ingot", TechExpansion.tab_resources, 64)
  val silver_ingot: TEItem = new TEItem("silver_ingot", TechExpansion.tab_resources, 64)

  def register(): Unit = {
    GameRegistry.register(copper_ingot)
    OreDictionary.registerOre("ingotCopper", copper_ingot)
    GameRegistry.register(aluminum_ingot)
    OreDictionary.registerOre("ingotAluminum", aluminum_ingot)
    GameRegistry.register(tin_ingot)
    OreDictionary.registerOre("ingotTin", tin_ingot)
    GameRegistry.register(nickel_ingot)
    OreDictionary.registerOre("ingotNickel", nickel_ingot)
    GameRegistry.register(lead_ingot)
    OreDictionary.registerOre("ingotLead", lead_ingot)
    GameRegistry.register(silver_ingot)
    OreDictionary.registerOre("ingotSilver", silver_ingot)
    GameRegistry.register(new CoFHTest())
  }

  def renders(): Unit = {
    RenderUtils.register(copper_ingot, "resources")
    RenderUtils.register(aluminum_ingot, "resources")
    RenderUtils.register(tin_ingot, "resources")
    RenderUtils.register(nickel_ingot, "resources")
    RenderUtils.register(lead_ingot, "resources")
    RenderUtils.register(silver_ingot, "resources")
  }
}
